src/engine/basics/Settings.cxx

```diff
@@ -64,6 +64,8 @@ void Settings::parse_args(int argc, char** argv) {
     if(patch[key].is_discarded()) {
       patch[key] = value;
     }
+
+    ++argv;
```

Comment:
I couldn't understand why we did add this here ? 
if you could evaluate more about this that would be perfect

Reply:
This is a bugfix. In case there is more than one parameter passed in the settings, the second parameter was not correctly parsed.

E.g. pass in the command parameters:
--UI.NewUI true --Game.Language fr